example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -54,9 +54,11 @@ public void onCreate() {
     super.onCreate();
     SoLoader.init(this, /* native exopackage */ false);
     if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
-      // If you opted-in for the New Architecture, we load the native entry point for this app.
+      // If you opted-in for the New Architecture, we load the native entry point for
+      // this app.
       DefaultNewArchitectureEntryPoint.load();
     }
-    ReactNativeFlipper.initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
+    // ReactNativeFlipper.initializeFlipper(this,
+    // getReactNativeHost().getReactInstanceManager());
```

Comment:
Esta é a liña importante, o de arriba é formateo automático de Visual Code Studio. Parece ser que para compilar .apks de release hay que comentar esta liña, ReactNativeFlipper está dentro da carpeta **debug/** e en na variant de release peta porque non o encontra. 

Non sei si nos convén mirar de arreglar esto ou deixalo comentado, ao final é unha liña dunha clase utilizada para debug dos exemplos que metín. Ademáis basta con comentala para sacar .apks de release e pasarllos a M para validar

Reply:
Non entendo este cambio, a pr suponse que tiña que ter unicamente o cambio da versión do sdk.

Reply:
Non se poden sacar .apks de release con esta liña de código porque ReactNativeFlipper é una clase dentro de **debug/**

Reply:
Non teño claro como funciona, que é o que te estás cargando exactamente? Que fai esa liña? 
Entendo que podes sacar un apk e logo deixalo como estaba.

Reply:
Estiven investigando un pouco por ahí e parece que esta clase axuda a debuggear mellor aplicacións de RN en IDEs como Visual Code Studio. Penso que non lle debería de dar tantas voltas e que o indicado é desfacer este cambio.